src/main/resources/templates/error.html

```diff
@@ -0,0 +1,8 @@
+<!DOCTYPE html>
```

Comment:
Added an html error page so now when someone goes to 127.0.0.1/ (any random URL you haven't mapped it shows this page)